guide/Community_Backlog.md

```diff
@@ -0,0 +1,165 @@
+
+- Only put GitHub issues in the project board, **do not add pull requests as they may cause duplication of context**
+- Make sure an issue passes through every column as they progress through each phase, that way the timing and context is documented, and no steps are missed
+
+### Board phases
+
+Board phases are represented by columns on GitHub. The following sections describe what process and what output occurs during their respective phases.
+
+1. [To do](#1-to-do)
+2. [Design queue](#2-design-queue)
+3. [Design intent](#3-design-intent)
+4. [Design review](#4-design-review)
+5. [Story writing](#5-story-writing)
+6. [Dev queue](#6-dev-queue)
+7. [In development](#7-in-development)
+8. [Ready for QA](#8-ready-for-qa)
+9. [In QA](#9-in-qa)
+10. [Ready for release](#10-ready-for-release)
+11. [Released](#11-released)
+
+#### 1. To do
+
+No work begins at this phase.
+
+Our reviewed backlog. Items are moved into this column when they are ready to be tracked.
+
+#### 2. Design queue
+
+No work is done at this phase.
+
+Items are moved from _To do_ to this column when they are ready to be picked up by a designer.
+
+#### 3. Design intent
+
+A designer pulls in a ticket from the top of the _Design queue_ and deliberates on best practices, branding, accessibility, and consistency for the ticket. Developers may stand by for consultation.
+
+Output:
+
+- During this phase, designers research the functionality, appearance, and intent for the ticket. They also establish preliminary designs.
+- Research material and any preliminary designs should be linked/attached to the ticket.
+
+#### 4. Design review
+
+Design leads and DPA Designers review final design. During a review, we determine:
+
+- Whether it is a pattern we want to encourage
+- Whether it is a core or community component
+- How it can scale between teams/tribe (i.e. reusability)
+
+If approved, they can go into _Story writing_ to be further refined.
+
+Once the design leads review the intent, DPA Designers and DPA Developers must sign off on the component’s viability based on the component criteria as outlined in our [contributing guide](../.github/CONTRIBUTING.md#designer-guide).
+
+Status of the review should be documented in the ticket.
+
+Output:
+
+- **DPA Design Approved** label is applied to ticket
+
+#### 5. Story writing
+
+A DPA member or assignee defines the acceptance criteria for the ticket.
+
+- Working with the designer/developer, the ticket is in the process of being defined
+- The person writing user stories for the ticket must fill out the required sections in the ticket template
+- Defining the acceptance criteria, functionality, design, and code notes and details
+
+Output:
+
+- The ticket has a written story
+- **DPA Developer Approved** label is applied
+- Ticket is moved to _Dev Queue_
+
+#### 6. Dev queue
+
+No work is done at this phase.
+
+Items are moved from Story Writing to this column when they are ready to be picked up by a developer and designer.
+
```

Comment:
Just a question.... should it be "by a developer and designer" or  "by a developer or designer". Does it have to be ready for both developer and designer in order for the story to be moved to this column?

Reply:
You are right, it should be developer `or` designer. I can update this.